tests/unit-tests/ambiguous/__snapshots__/arrow-parens-avoid.test.ts.snap

```diff
@@ -986,7 +986,7 @@ oops => {};
 exports[`ambiguous > config > arrowParens: "avoid" > (oops) => {} > with semi, without newline > it formats ../cases/gjs/simple.gjs 1`] = `
 "<template>
   what
-</template>
+</template>;
 oops => {};
```

Comment:
I'm not sure why this happens... Looking into it

Reply:
yes, somehow it ended up inside an ExpressionStatement...

Reply:
Probably because of the ambiguity. 🙃

Reply:
Right, i fixed it